lib/server/imgix.js

```diff
@@ -36,7 +34,10 @@ export const imgixResolver = async (parent, { input }, context) => {
       domain: process.env.IMGIX_WEB_PROXY_HOSTNAME,
       secureURLToken: process.env.IMGIX_WEB_PROXY_TOKEN,
     });
-    const twUrl = fullParentFSFile.link();
+    const fullParentFSFileDocument = FSCollection.findOne({
+      _id: parentFSDocumentId,
+    });
+    const twUrl = fullParentFSFileDocument.link();
```

Comment:
sweet! one nice thing while we're at it would be to delete lines 51 to 57 :) 

making changes in submodules is always a pain, so its nice to do things in one fell swoop.

Reply:
should we still return `imgixUrl`?